src/Modal/marketing-modal.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'MarketingModal'
+type: 'component'
+components:
+- MarketingModal
+- ModalDialog
+categories:
+- Overlays
+status: 'New'
+designStatus: 'Done'
+devStatus: 'Done'
+notes: |
+
+---
+
+The composition of `ModalDialog` for marketing style scenarios. This component passes through all of its props to an underlying `ModalDialog` component providing some limited customization. If you have unique needs, use the `ModalDialog` compound component family directly.
+
+```jsx live
+() => {
+  const [isOpen, open, close] = useToggle(false);
+  return (
+    <>
+      <Button variant="outline-primary" onClick={open}>Open marketing modal</Button>
+
+      <MarketingModal
+        title="My dialog"
```

Comment:
I think this should not be here? Unless it has a purpose I'm not aware of. I'm not seeing this title actually appear anywhere.

Reply:
Dialogs must be labeled for a11y reasons. It's applied to `aria-label` on the dialog